pgbouncer/datadog_checks/pgbouncer/metrics.py

```diff
@@ -55,7 +55,7 @@
 }
 
 DATABASES_METRICS = {
-    'descriptors': [('name', 'name')],
+    'descriptors': [('name', 'name'), ('name', 'db')],
```

Comment:
Sorry if that's a stupid question but what's the difference between the column `name` and `database` in `SHOW DATABASES`?

Reply:
@FlorentClarret , `show databases` is not supported in PostgreSQL. Do you use \l or \l+?

Reply:
What this is trying to do is just to export the same metric "name" into another metric output called "db" to allow us to apply formula in the dashboard.

Reply:
`show databases` is the query we perform for these metrics: https://github.com/DataDog/integrations-core/blob/master/pgbouncer/datadog_checks/pgbouncer/metrics.py#L64

Docs here I think: https://www.pgbouncer.org/usage.html

I'm not a pgbouncer expert and I see there's another column called `database`, but I don't understand the difference with `name` and I'm wondering if adding `db` would not be confusing here